### DISCUSSION TITLE
Issue with choices

### DISCUSSION THREAD

**Discussion by EriksBlue** [dialogic-godot/dialogic#1791] [Q&A] [answered]
For Godot 4, im trying to place choices in the timeline and am getting the yellow triangle "this event needs a specific parent event", I have not found any information in the documentation that can help me. Can anyone give me a idea of what im supposed to do?

**Comment by CakeVR** [comment, marked as answer]
Here is an example on how you can do Choices in Dialogic 2:
![image](https://github.com/coppolaemilio/dialogic/assets/65981767/b69ba294-214e-4b7c-825b-64c2d05b8854)

The text editor will show this:
```
What's your favourite tea?
- Black tea
- Green tea
- Oolong tea
```

If I would have to guess, your problem is caused by adding a Choice Event and then Dialogic 2 selected that added Event. 
When it tried to add another Choice Event, it attempted to nest the Choice Event inside the now selected Event.

**Reply by EriksBlue** [reply]
Yes thank you, I think what happened was I placed a Background change just before the Choice and it didn't nest properly.
